app/controllers/admin/organization-invites/new.js

```diff
@@ -7,6 +7,7 @@ export default Controller.extend({
   flashMessages: service(),
   store: service(),
 
+  modelName: 'organizationInvite',
```

Comment:
Not sure I'm a huge fan of this pattern.  The mixin needs to determine a dynamic model on the route and we don't have a common identifier such as `model`.  Have any thoughts?

Reply:
Not sure if there's another way to go about it.

To me, calling the main model just `model` isn't great either. It lacks explicitness. 

We expected to require this. I just wish there was a way to extend a mixin while explicitly providing an option for it, kind of like elixir's `use X, option: something`. This line lacks explicitness to.

I'm also not sure I like the fact that we have to use the mixin in a route and then specify a model name in the controller.That part, we can change, though, with the way the mixin works, so could you move the model name into route?

@joshsmith What do you think about the other stuff?

Maybe naming can help?

We could call the mixin `RollbackModelOnNavigatingAway`. It's verbose, but also provides clarity for itself, as well as for the `modelName` property we could use on the route.

We could also make the modelName into an alias, as well as call it `rollbackModel` to add further clarity. In this specific case:

```javascript
export default Route.extend(RollbackModelOnNavigatingAway, {
  rollbackModel: alias('controller.organizationInvite')
});
```

Reply:
@begedin I moved the name to the route.  Lmk what you still think about the naming...I can change it if you would like!